website/content/docs/enterprise/namespaces.mdx

```diff
@@ -82,7 +82,7 @@ namespace. This data includes the following:
 
 - Secret Engines
 - Auth Methods
-- Policies
+- Policies (including Password Policies)
```

Comment:
I wonder if it's better to break these into separate and explicit entries
```suggestion
- ACL, EGP, and RGP Policies 
- Password Policies 
```

Reply:
Yep